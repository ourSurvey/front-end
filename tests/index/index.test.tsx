import { render, screen } from "@testing-library/react";
import Home from "../../pages/index";

describe("Home", () => {
  it("renders a button", () => {
    const { container } = render(<Home />);

    const heading = screen.getByText("이메일");

    expect(heading).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
