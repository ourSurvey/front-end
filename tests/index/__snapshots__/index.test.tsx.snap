// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders a button 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="formContainer"
    >
      <div
        class="headerName"
      >
        <div
          class="arrow"
        />
        <span
          class="name"
          style="margin-left: 20px; margin-right: 20px;"
        >
          로그인
        </span>
        <div
          class="arrow"
        />
      </div>
      <form
        novalidate=""
      >
        <div
          class="inputContainer"
        >
          <label
            for="이메일-input"
          >
            이메일
          </label>
          <br />
          <input
            id="이메일-input"
            name="이메일"
            type="text"
          />
        </div>
        <div
          class="inputContainer mt30"
        >
          <label
            for="비밀번호-input"
          >
            비밀번호
          </label>
          <br />
          <input
            id="비밀번호-input"
            name="비밀번호"
            type="text"
          />
        </div>
        <div
          class="mt30"
        >
          <button
            style="height: 50px; width: 100%; background-color: rgb(0, 102, 217); color: rgb(255, 255, 255); border-radius: 10px; font-family: Pretendard; font-style: normal; font-weight: 700; font-size: 14px; border: 0px; outline: 0; padding-top: 16.5px; padding-bottom: 16.5px;"
          >
            로그인
          </button>
        </div>
        <div
          class="center"
        >
          <span>
            비밀번호 찾기
          </span>
        </div>
      </form>
    </div>
  </div>
</div>
`;
